docs/src/content/docs/commands/KEYS.md

```diff
@@ -13,57 +13,65 @@ KEYS pattern
 
 ## Parameters
 
-- `pattern`: A string that represents the pattern to match against the keys in the DiceDB database. The pattern can include special glob-style characters:
-  - `*` matches any number of characters (including zero).
-  - `?` matches exactly one character.
-  - `[abc]` matches any one of the characters inside the brackets.
-  - `[a-z]` matches any character in the specified range.
+| Parameter       | Description                                      | Type    | Required |
+|-----------------|--------------------------------------------------|---------|----------|
+| `pattern`           | A string representing the pattern to match against the keys in DiceDB databse.| String  | Yes      |
 
-## Return Value
+Supported glob-style characters:
+   - `*` matches any number of characters (including zero).
+   - `?` matches exactly one character.
+   - `[abc]` matches any one of the characters inside the brackets.
+   - `[a-z]` matches any character in the specified range.
 
-The `KEYS` command returns an array of strings, where each string is a key that matches the specified pattern. If no keys match the pattern, an empty array is returned.
+## Return values
+
+| Condition                                      | Return Value                                      |
+|------------------------------------------------|---------------------------------------------------|
+| Command is successful                          | Array of strings, where each string is key that matches the specified pattern.|
+| Command is successful but key not found        | `(empty list or set)`  |
+| Syntax or specified constraints are invalid    | error   |
 
 ## Example Usage
 
 ### Basic Example
 
 ```plaintext
-DiceDB> SET key1 "value1"
+127.0.0.1:7379> SET key1 "value1"
 OK
-DiceDB> SET key2 "value2"
+127.0.0.1:7379> SET key2 "value2"
 OK
-DiceDB> SET anotherkey "value3"
+127.0.0.1:7379> SET anotherkey "value3"
 OK
-DiceDB> KEYS key*
+127.0.0.1:7379> KEYS key*
 1) "key1"
 2) "key2"
 ```
 
 ### Using Wildcards
 
 ```plaintext
-DiceDB> SET key1 "value1"
+127.0.0.1:7379> SET key1 "value1"
 OK
-DiceDB> SET key2 "value2"
+127.0.0.1:7379> SET key2 "value2"
 OK
-DiceDB> SET key3 "value3"
+127.0.0.1:7379> SET key3 "value3"
 OK
-DiceDB> KEYS key?
-1) "key1"
-2) "key2"
-3) "key3"
+127.0.0.1:7379> KEYS key?
```

Comment:
@apoorvyadav1111 We already have that in the examples. Look at this.

Reply:
Yes, my suggestion was to add an example using \ such as below.
```
127.0.0.1:7379> SET key1 "value1"
DiceDB> SET key1 "value1"
OK
127.0.0.1:7379> SET key2 "value2"
DiceDB> SET key? "value2"
OK
127.0.0.1:7379> SET key3 "value3"
DiceDB> SET key* "value3"
OK
127.0.0.1:7379> KEYS key?
1) "key3"
2) "key*"
3) "key?"
DiceDB> KEYS key\?
1) "key?"
```